images/macos/software-report/SoftwareReport.Common.psm1

```diff
@@ -129,4 +130,320 @@ function Build-OSInfoSection {
     $output += New-MDHeader "macOS $version info" -Level 1
     $output += New-MDList -Style Unordered -Lines $parsedSystemInfo -NoNewLine
     return $output
+}
+
+function Get-PHPVersion {
+    $PHPVersion = Run-Command "php --version" | Select-Object -First 1 | Take-Part -Part 0,1
+    return $PHPVersion
+}
+
+function Get-NodeVersion {
+    $nodeVersion = Run-Command "node --version"
+    return "Node $nodeVersion"
+}
+
+function Get-PythonVersion {
+    $pythonVersion = Run-Command "python --version"
+    return $pythonVersion
+}
+
+function Get-Python3Version {
+    $python3Version = Run-Command "python3 --version"
+    return $python3Version
+}
+
+function Get-RubyVersion {
+    $rubyVersion = Run-Command "ruby --version" | Take-Part -Part 1
+    return "Ruby $rubyVersion"
+}
+
+function Get-PHPVersion {
+    $PHPVersion = Run-Command "php --version" | Select-Object -First 1 | Take-Part -Part 0,1
+    return $PHPVersion
+}
+
+function Get-JuliaVersion {
+    $juliaVersion = Run-Command "julia --version" | Take-Part -Part 0,2
+    return $juliaVersion
+}
+
+function Get-BundlerVersion {
+    $bundlerVersion = Run-Command "bundle --version"
+    return $bundlerVersion
+}
+
+function Get-CarthageVersion {
+    $carthageVersion = Run-Command "carthage version" -SuppressStderr
+    return "Carthage $carthageVersion"
+}
+
+function Get-CocoaPodsVersion {
+    $cocoaPodsVersion = Run-Command "pod --version"
+    return "CocoaPods $cocoaPodsVersion"
+}
+
+function Get-HomebrewVersion {
+    $homebrewVersion = Run-Command "brew --version" | Select-Object -First 1
+    return $homebrewVersion
+}
+
+function Get-NPMVersion {
+    $NPMVersion = Run-Command "npm --version"
+    return "NPM $NPMVersion"
+}
+
+function Get-YarnVersion {
+    $yarmVersion = Run-Command "yarn --version"
+    return "Yarn $yarmVersion"
+}
+
+function Get-NuGetVersion {
+    $nugetVersion = Run-Command "nuget help" | Select-Object -First 1 | Take-Part -Part 2
+    return "NuGet $nugetVersion"
+}
+
+function Get-CondaVersion {
+    $condaVersion = Invoke-Expression "conda --version"
+    return "Mini $condaVersion"
```

Comment:
Miniconda without space
![image](https://user-images.githubusercontent.com/8777209/103093142-934f5800-460a-11eb-97a9-185a348bf2f6.png)

Reply:
Thank you, fixed it